src/flows/OnboardingFlow/steps/Welcome.tsx

```diff
@@ -5,8 +5,18 @@ import Button from 'components/core/Button/Button';
 import colors from 'components/core/colors';
 import Spacer from 'components/core/Spacer/Spacer';
 import FullPageCenteredStep from 'flows/shared/components/FullPageCenteredStep/FullPageCenteredStep';
+import CoolIntro from 'scenes/CoolIntro/CoolIntro';
 
 function Welcome({ next }: WizardContext) {
+  // return <WelcomeText next={next} />;
+  return <CoolIntro next={next}></CoolIntro>;
```

Comment:
fix?

Reply:
Done - removed commented line and renamed CoolIntro to WelcomeAnimation lol